framework/helpers/BaseHtml.php

```diff
@@ -430,11 +430,22 @@ public static function mailto($text, $email = null, $options = [])
      * the attributes of the resulting tag. The values will be HTML-encoded using [[encode()]].
      * If a value is null, the corresponding attribute will not be rendered.
      * See [[renderTagAttributes()]] for details on how attributes are being rendered.
+     * @since 2.0.12 It is possible to pass the "srcset" option as an array which keys are
+     * descriptors and values are URLs. All URLs will be processed by [[Url::to()]].
```

Comment:
@cebe you can have multiple `@since` tags according to the [docs](https://phpdoc.org/docs/latest/references/phpdoc/tags/since.html) to specify what happened in which patch. Having a single `@since` doesn't mean that this element was introduced in this patch.